services/app-web/src/constants/routes.ts

```diff
@@ -164,6 +167,7 @@ const PageTitlesRecord: Record<RouteT | 'UNKNOWN_ROUTE', string> = {
     SUBMISSIONS_REVISION: 'Submission revision',
     SUBMISSIONS_SUMMARY: 'Submission summary',
     SUBMISSIONS_QUESTIONS_AND_ANSWERS: 'Q&A',
+    SUBMISSIONS_RATE_QUESTIONS_AND_ANSWERS: 'Q&A',
```

Comment:
Let's get different page titles here - these are announced by the screenreader. Seems like it could be `Rate Q&A`